### ISSUE TITLE
TexStudio crashes when starting

### ISSUE THREAD

**Issue by jcasale** [texstudio-org/texstudio#3858]
### Environment
- TeXstudio: 4.8.5beta1
- Qt: unable to check
- OS: Windows 11
- TeX distribution: <miktex

### Expected behavior
When starting TexStudio, it should not crash.

### Actual behavior
I am no longer able to start TexStudio, I have removed the `~AppData\Roaming\texstudio` directory to revert any potential corrupt settings, I have uninstalled and reinstalled.

I have several crash dump files if that would be of any help.

### How to reproduce
I cannot offer any way to reproduce this, it was working for the last 12 hours. However, after resetting the theme, I am unable to start it without a crash.

**Comment by jcasale** [comment]
Rolling back to 4.8.5alpha1 has resolved the issue, strange as I used it through the weekend.

**Comment by sunderme** [comment]
you need to be a bit more precise on what you did.
The main difference between alpha1 and beta1 is the use of qt6.8.

**Comment by jcasale** [comment]
I wish had more to offer. I installed it the morning it was released and used it for a couple of days. I did see a few crashes when starting it periodically.

I then tried to resume work yesterday and it would crash every time I would try to start it.

I originally thought it was a settings issue, but removing the `~AppData\Roaming\texstudio` directory did not help, I had to revert to the previous version.

There looks like some problems with qt6.8, but I am not sure if they are related:

- https://github.com/msys2/MINGW-packages/issues/22308
- https://bugreports.qt.io/browse/QTBUG-130597

Would a mini dump help?
